### PR TITLE
Feature/iot 1416

### PR DESCRIPTION
**Pull request by Fritsen**

### CODE REVIEW

File: src/config/configuration.ts [hunk 1]
```diff
@@ -39,15 +39,15 @@ export default (): any => {
         email: {
             host: process.env.EMAIL_HOST || "smtp.ethereal.email",
             port: process.env.EMAIL_PORT || 587,
-            user: process.env.EMAIL_USER || "ara.kertzmann8@ethereal.email",
```
Comment (fcv-iteratorIt) [inline src/config/configuration.ts:42]: Was this changed on purpose?

**Review by fcv-iteratorIt — CHANGES_REQUESTED** [review]
Some minor changes and suggestions

File: src/controllers/admin-controller/data-target.controller.ts [hunk 2]
```diff
@@ -188,4 +213,47 @@ export class DataTargetController {
...
+    @ApiOperation({
+        summary: "Update the OpenDataDkRegistered to true, for the given OrganizationId - to stop showing the dialog for sending ODDK-mail on creation of new datatargets",
+    })
+    @ApiNotFoundResponse()
+    async updateOpenDataDkRegistered(
+        @Req() req: AuthenticatedRequest,
+        @Param("organizationId", new ParseIntPipe()) organizationId: number
+    ): Promise<boolean> {
+        try {
+            await this.organizationService.updateOpenDataDkRegistered(organizationId, req.user.userId);
```
[the remark this answers is not in the document]
Comment (Fritsen) [reply src/controllers/admin-controller/data-target.controller.ts:239]: I'm fairly sure it wouldn't. The flag is to keep track of whether a mail has been sent to OpenDataDK. Once the mail is sent, we can't really take it back...

There _is_ an option to "never show this again", without sending the mail. In theory we could - sometime in the future - allow users to then change their mind, and reset the flag somehow. But I don't think we'll actually ever make that :)

File: src/entities/opendatadk-push-data-target.entity.ts
[the remark this answers is not in the document]
Comment (Fritsen) [reply src/entities/opendatadk-push-data-target.entity.ts:1]: Fair

File: src/config/configuration.ts (hunk 1 above)
Comment (Fritsen) [reply src/config/configuration.ts:42]: Yes.

ethereal.email uses auto-generated accounts, and are only used to test mails during development. The account in the config did not work anymore, so I auto-generated a new one, and left it in the config, to make it (very slightly) easier for the next task

**Pull request merged**